src/client_metrics.rs

```diff
@@ -177,14 +177,14 @@ pub struct MetricsMetadata {
 pub struct MetricSample {
     pub value: f64,
     #[serde(default, skip_serializing_if = "Option::is_none")]
-    pub labels: Option<HashMap<String, String>>,
```

Comment:
do we want to consider the same change for pub toggles: HashMap<String, ToggleStats> in MetricBucket?